sdk/tools/spec2def/spec2def.c

```diff
@@ -49,6 +49,7 @@ char *pszSourceFileName = NULL;
 char *pszDllName = NULL;
 char *gpszUnderscore = "";
 int gbDebug;
+unsigned guOsVersion = 0x502;
```

Comment:
Don't we have the _WIN32_WINNT macros available also for the host-tools? (you would specify an explicit _WIN32_WINNT_WS03 here for example).

Reply:
We don't